csrankings-r.csv

```diff
@@ -71,6 +71,7 @@ Radu Prodan,University of Klagenfurt,http://www.itec.aau.at/~radu,ELx8lB8AAAAJ
 Radu Sion,Stony Brook University,https://www.cs.stonybrook.edu/people/faculty/RaduSion,cn-X2AIAAAAJ
 Radu Stoleru,Texas A&M University,http://faculty.cs.tamu.edu/stoleru,beAeG20AAAAJ
 Radu Teodorescu,Ohio State University,http://web.cse.ohio-state.edu/~teodores,IKz0hvkAAAAJ
+Radu-Cristian Curticapean,IT University of Copenhagen,https://pure.itu.dk/portal/da/persons/raducristian-curticapean(53c45a58-4c3c-424c-80a2-c39e733fd7c3).html,NOSCHOLARPAGE

```

Comment:
Google Scholar page is here: https://scholar.google.com/citations?user=5MeyUVQAAAAJ&hl=en&oi=sra; please update.

Reply:
fixed this, thanks